src/gossip/pull_request.zig

```diff
@@ -158,11 +159,16 @@ pub const GossipPullFilterSet = struct {
     }
 
     /// returns a list of GossipPullFilters and consumes Self by calling deinit.
-    pub fn consumeForGossipPullFilters(self: *Self, alloc: std.mem.Allocator, rand: std.Random, max_size: usize) error{OutOfMemory}!ArrayList(GossipPullFilter) {
+    pub fn consumeForGossipPullFilters(
+        self: *Self,
+        allocator: std.mem.Allocator,
+        rng: std.Random,
```

Comment:
For consistency with the type names, and to avoid confusion, I usually like to name implementations `rng`, and the interface `random`.

```zig
rng: DefaultPrng
random: Random
```

Reply:
makes sense